js/src/lib/inferenceSnippets/servePython.ts

```diff
@@ -0,0 +1,52 @@
+import type { PipelineType, ModelData } from "../interfaces/Types";
+import { getModelInputSnippet } from "./inputs";
+
+export const bodyZeroShotClassification = (model: ModelData): string =>
+	`output = query({
+    "inputs": ${getModelInputSnippet(model)},
+    "parameters": {"candidate_labels": ["refund", "legal", "faq"]},
+})`;
+
+export const bodyBasic = (model: ModelData): string =>
+	`output = query({
+	"inputs": ${getModelInputSnippet(model)},
+})`;
+
+export const pythonSnippetBodies:
+	Partial<Record<keyof typeof PipelineType, (model: ModelData) => string>> =
+{
+	"zero-shot-classification": bodyZeroShotClassification,
```

Comment:
How do you feel about keeping same ordering as in https://github.com/huggingface/huggingface_hub/blob/main/js/src/lib/interfaces/Types.ts for easier reading?

Reply:
Sure thing :)

Reply:
Done !